common/src/main/java/bisq/common/application/ApplicationVersion.java

```diff
@@ -17,20 +17,16 @@
 
 package bisq.common.application;
 
-import bisq.common.util.FileUtils;
 import bisq.common.util.Version;
 
-import java.io.IOException;
-
 public class ApplicationVersion {
     private static Version version;
 
     public static Version getVersion() {
         if (version == null) {
             try {
-                String versionString = FileUtils.readStringFromResource("version.txt");
```

Comment:
Thanks @djing-chan, I'll fix this and remove the version.txt in common in a tick!

Regarding the binary issue on MacOS, that sounds unrelated to this change.. What it can happen is a cache issue, if you are using Intellij to test this make sure you invalidate caches and restart. On terminal, a gradlew clean should do. 

I have a macbook here I'll give it a test just in case in a few moments, thanks!